<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCMBProjectManagerMainWindow</class>
 <widget class="QMainWindow" name="qtCMBProjectManagerMainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/cmb/Icon.png</normaloff>:/cmb/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="PointsButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Launch Points Builder</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/cmb/PointsBuilderIcon.png</normaloff>:/cmb/PointsBuilderIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PointsArrow">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Resources/Resources.qrc">:/cmb/Arrow.png</pixmap>
      </property>
      <property name="buddy">
       <cstring>SceneButton</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SceneButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Launch Scene Builder</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/cmb/SceneBuilderIcon.png</normaloff>:/cmb/SceneBuilderIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SceneArrow">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Resources/Resources.qrc">:/cmb/Arrow.png</pixmap>
      </property>
      <property name="buddy">
       <cstring>ModelButton</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ModelButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Launch Model Builder</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/cmb/ModelBuilderIcon.png</normaloff>:/cmb/ModelBuilderIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ModelArrow">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Resources/Resources.qrc">:/cmb/Arrow.png</pixmap>
      </property>
      <property name="buddy">
       <cstring>SimButton</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SimButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Launch Simulation Builder</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/cmb/MeshViewerIcon.png</normaloff>:/cmb/MeshViewerIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionServerConnect"/>
    <addaction name="actionServerDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionServerConnect">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/cmb/pqConnect24.png</normaloff>:/cmb/pqConnect24.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="statusTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionServerDisconnect">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/cmb/pqDisconnect24.png</normaloff>:/cmb/pqDisconnect24.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="statusTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
